pkg/types/relayer.go

```diff
@@ -141,6 +140,12 @@ type EVMService interface {
 	// GetTransactionFee retrieves the fee of a transaction in wei from the underlying chain
 	GetTransactionFee(ctx context.Context, transactionID IdempotencyKey) (*evm.TransactionFee, error)
 
+	// WriteReport writes a transaction against a keystone forwarder contract
+	SubmitTransaction(ctx context.Context, txRequest evm.SubmitTransactionRequest) (*evm.TransactionResult, error)
```

Comment:
```suggestion
	// SubmitTransaction writes a transaction against a keystone forwarder contract
	SubmitTransaction(ctx context.Context, txRequest evm.SubmitTransactionRequest) (*evm.TransactionResult, error)
```
Also is this comment even necessary?